src/ai/backend/web/server.py

```diff
@@ -590,6 +590,11 @@ async def server_main(
     cors.add(app.router.add_route('POST', '/func/{path:.*$}', web_handler))
     cors.add(app.router.add_route('PATCH', '/func/{path:.*$}', web_handler))
     cors.add(app.router.add_route('DELETE', '/func/{path:.*$}', web_handler))
+    cors.add(app.router.add_route('GET', '/{name:flow}/{path:.*$}', web_handler))
+    cors.add(app.router.add_route('PUT', '/{name:flow}/{path:.*$}', web_handler))
+    cors.add(app.router.add_route('POST', '/{name:flow}/{path:.*$}', web_handler))
+    cors.add(app.router.add_route('PATCH', '/{name:flow}/{path:.*$}', web_handler))
+    cors.add(app.router.add_route('DELETE', '/{name:flow}/{path:.*$}', web_handler))
```

Comment:
I think we need a precaution step to prevent overlapping of the flow name and `"func"`.  The best way to do is to prefix the URL routes, e.g., `'/pipeline/{name:flow}/{path:.*$}'`.